nautobot/core/views/mixins.py

```diff
@@ -957,7 +958,19 @@ def send_bulk_delete_objects_to_job(self, request, pk_list, model, delete_all):
         if filterset_class := lookup.get_filterset_for_model(model):
             filter_query_params = normalize_querydict(request.GET, filterset=filterset_class())
         else:
-            filter_query_params = None
+            filter_query_params = {}
+
+        # Discarding non-filter query params
+        new_filter_query_params = {}
+
+        for key, value in filter_query_params.items():
+            try:
+                filtering.get_filterset_field(filterset_class(), key)
+                new_filter_query_params[key] = value
+            except core_exceptions.FilterSetFieldNotFound:
+                self.logger.warning(f"Query parameter `{key}` not found in `{filterset_class}`, discarding it")
```

Comment:
This can probably be a debug rather than a warning, as it's likely to occur during normal operation.

Same comment below.